src/test/java/com/microsoft/sqlserver/jdbc/fedauth/FedauthTest.java

```diff
@@ -380,41 +424,26 @@ public void testAccessTokenCallbackClassConnection() throws Exception {
     @Test
     public void testAccessTokenCache() {
         try {
-            Set<IAccount> accountsInCache = fedauthPcaApp.getAccounts().join();
-            assertTrue(null != accountsInCache && !accountsInCache.isEmpty());
-
-            IAccount account = getAccountByUsername(accountsInCache, azureUserName);
-            assertNotNull(account);
-
-            SilentParameters silentParameters = SilentParameters
-                    .builder(Collections.singleton(spn + "/.default"), account).build();
+            SilentParameters silentParameters = SilentParameters.builder(Collections.singleton(spn + "/.default"))
+                    .build();
 
             // this will fail if not cached
-            CompletableFuture<IAuthenticationResult> future = fedauthPcaApp.acquireTokenSilently(silentParameters);
+            CompletableFuture<IAuthenticationResult> future = fedauthClientApp.acquireTokenSilently(silentParameters);
             IAuthenticationResult authenticationResult = future.get();
             assertNotNull(authenticationResult.accessToken());
-            assertEquals(authenticationResult.account().username(), azureUserName);
+            assertTrue(authenticationResult.accessToken().equals(accessToken), accessToken.toString());

```

Comment:
Unnecessary toString().
```suggestion
            assertTrue(authenticationResult.accessToken().equals(accessToken), accessToken);
```